routes/users.js

```diff
@@ -28,9 +29,7 @@ router.post('/', validator(UserBodySchema), (req, res, next) => {
 
 router.delete('/:usrId', (req, res, next) => {
```

Comment:
You don't check here if a user that you're deleting is current user. That means everyone can delete anyone.

Reply:
You're sending all user's data including password hash.